tools/pyproject.toml

```diff
@@ -17,9 +25,9 @@ tabulate = "^0.9.0"
 [tool.poetry.group.dev.dependencies]
 black = {extras = ["d"], version = "^23.3.0"}
 pdbpp = "^0.10.3"
-mypy = "^1.7.0"
 types-requests = "^2.31.0.10"
 types-tabulate = "^0.9.0.3"
+mypy = "1.6.1"
```

Comment:
mypy 1.7.0 was having issues installing w/poetry + python 3.11 (it couldn't solve the dependency constraint for reasons that remain unclear). Rather than burn more cycles I've downgraded to 1.6.1 for now as 1.7.0 is pretty fresh (last two weeks).

Reply:
Thanks, I hit this issue.